tests/conftest.py

```diff
@@ -1,42 +1,83 @@
 # SPDX-License-Identifier: MIT
 
+import os
+import os.path
+import shutil
+import stat
+import tempfile
+
 import pytest
 
 
-DUMMY_PYPROJECT = '''
-[build-system]
-requires = ['flit_core']
-build-backend = 'flit_core.buildapi'
-'''.strip()
+def pytest_addoption(parser):
+    parser.addoption('--run-integration', action='store_true', help='run the integration tests')
+    parser.addoption('--only-integration', action='store_true', help='only run the integration tests')
+
+
+def pytest_collection_modifyitems(config, items):
+    skip_int = pytest.mark.skip(reason='integration tests not run')
+    skip_other = pytest.mark.skip(reason='only integration tests are run')
+
+    for item in items:
+        if os.path.basename(item.location[0]) == 'test_integration.py':
```

Comment:
maybe better to have an integration folder rather than a single file 🤔

Reply:
There is nothing right now warranting an integration folder, when there is, this can be change. Making a `integration` folder for these tests would be an premature optimization.